packages/auth/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@aerogearservices/auth",
```

Comment:
should this just be `@aerogear` based on the latest discussion on the mailing list.